mesonbuild/interpreter/interpreter.py

```diff
@@ -616,7 +616,10 @@ def func_import(self, node: mparser.BaseNode, args: T.Tuple[str],
         expect_unstable = False
         # Some tests use "unstable_" instead of "unstable-", and that happens to work because
         # of implementation details
-        if modname.startswith('unstable'):
+        if modname.startswith(('unstable_', 'unstable-')):
```

Comment:
This hunk is in the wrong commit. It belongs in "move handling of module stability to interpreter" but it's in "deprecate the ability import unstable modules as `unstable_*`"

It's correct to check both "unstable-" and "unstable_" in the original implementation, rather than switching to that later on from "unstable".